example/hello-world/build.rs

```diff
@@ -1,10 +1,10 @@
 use g2h::BridgeGenerator;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // Simple approach with default settings
+    // Simple one-liner with automatic string enum support
     BridgeGenerator::with_tonic_build()
-        .build_prost_config()
-        .compile_protos(&["protos/hello-world.proto"], &["protos"])?;
+        .with_string_enums()
```

Comment:
```suggestion
        .build_prost_config()
        .with_string_enums()
```

we can do something like this, no?